modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -110,6 +134,13 @@ grpcurl -proto text-generation-inference/proto/generation.proto -d '{"requests":
 curl -ks <inference_endpoint_url>/v2/models/<model_name>/infer -d '{ "model_name": "<model_name>", "inputs": [{ "name": "<name_of_model_input>", "shape": [<shape>], "datatype": "<data_type>", "data": [<data>] }]}' -H 'Authorization: Bearer <token>'  <1>
 ----
 <1> You must add the `Authorization` header and specify a token value _only_ if you enabled token authorization when deploying the model.
+
```

Comment:
Not really relevant for this PR, but the above `grpcurl` command has an extra `-H 'mm-model-id`` argument which is not really required (and is going to be ignored)

Reply:
Addressed this one as well in the latest commit.

Reply:
@jbyrne-redhat